rules/linux/macos_creds_from_keychain.yml

```diff
@@ -0,0 +1,21 @@
+title: Credentials from Password Stores - Keychain
+id: b120b587-a4c2-4b94-875d-99c9807d6955
+status: experimental
+description: Detects passwords' dumps from Keychain
+author: Tim Ismilyaev, oscd.community
+date: 2020/10/19
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1555.001/T1555.001.md
+logsource:
+    category: process_creation
+    product: macos
+detection:
+    any_keychain_call:
+        ProcessName|startswith: 'security'
```

Comment:
Hello @tim1234567! Thank you for your contribution!
Could you please change 

```
        ProcessName|startswith: 'security'
```

to the

```
        ProcessName|endswith: '/security'
```

as the full path to the binary (which is going to be logged) is `/usr/bin/security`? 

Thanks!

Reply:
Hi @tim1234567!
Could you make the changes that were recommended to you above?
Thanks!

Reply:
Thanks, Dan and @e6e6e, for the feedback. I'll push changes in the next few days. Sorry for the delay.

Reply:
thanks!